src/components/WithForwardRef/withForwardRef.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'React';
+
+export interface WithForwardRefProps<T extends HTMLElement = HTMLElement> {
+  innerRef?: React.Ref<T>;
```

Comment:
I'm also not so sure about this name.
I know changing this name will cause a greater change for the button component,
but maybe something like `exposedRef` will be more readable?
And maybe instead of `withForwardRef` we can have `exposeRef()`?
I'm really not sure about this.
what do you think?